pkg/utils/utils.go

```diff
@@ -26,15 +26,22 @@ import (
 )
 
 func RandDate(rnd *rand.Rand) string {
-	return RandTime(rnd).Format("2006-01-02")
+	min := time.Date(1970, 0, 0, 0, 0, 0, 0, time.UTC).Unix()
+	max := time.Date(2024, 1, 0, 0, 0, 0, 0, time.UTC).Unix()
+
+	sec := rnd.Int63n(max-min) + min
+	return time.Unix(sec, 0).UTC().Format("2006-01-02")
 }
 
 func RandTime(rnd *rand.Rand) time.Time {
-	min := time.Date(1970, 1, 0, 0, 0, 0, 0, time.UTC).Unix()
-	max := time.Date(2024, 1, 0, 0, 0, 0, 0, time.UTC).Unix()
+	return time.Unix(rnd.Int63n(86400), rnd.Int63n(1000000000)).UTC()
+}
 
-	sec := rnd.Int63n(max-min) + min
-	return time.Unix(sec, 0).UTC()
+func RandTimeStamp(rnd *rand.Rand) time.Time {
+	min := time.Date(1970, 0, 0, 0, 0, 0, 0, time.UTC).UnixMilli()
+	max := time.Date(2024, 1, 0, 0, 0, 0, 0, time.UTC).UnixMilli()
+	milliSec := rnd.Int63n(max-min) + min
+	return time.UnixMilli(milliSec).UTC()
```

Comment:
What is the difference between RandTime and RandTimeStamp ?

Reply:
Time should be not more that nanosecond in day.